marimbabot_behavior/launch/test_behavior.launch

```diff
@@ -1,7 +1,6 @@
 <launch>
 <node name="behavior_node" pkg="marimbabot_behavior" type="behavior_node.py" output="screen"/>
 <node name="audio_from_lilypond" pkg="marimbabot_audio" type="audio_from_lilypond.py"/>
-<include file="$(find marimbabot_audio)/launch/midi.launch"/>
```

Comment:
Why did you remove this?

Reply:
This launchfile no longer exists. For testing, I needed to comment out this line. Preview and TTS still function.

Reply:
Ah okay!